iguana/launch_assetchains.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eo pipefail
+
+# You can now add delay line to pubkey.txt file
+source pubkey.txt
+overide_args="$@"
+seed_ip=`getent hosts zero.kolo.supernet.org | awk '{ print $1 }'`
+komodo_binary='./komodod'
```

Comment:
why do we assume that komodod is in the same folder with script?

Reply:
> why do we assume that komodod is in the same folder with script?

good point - should be set to expected path `~/komodo/src/komodod`

Reply:
i would assume komodod is in the path and not rely that people are cloning in home directory... i do not, for example

Reply:
Probably we can try to combine like a  :

`~/komodo/src/komodod || komodod`

to handle more possible setups

Reply:
Maybe this should be defined in a separate file like pubkey.txt (or komodods.json) to suit user setups?
This might also cater for assetchains using different `version/tree` as stated in the README, for example STBL and VOTE2020 compared to all other assetchains (i.e. in the event of chains using a different komodod).

Reply:
```
This might also cater for assetchains using different version/tree as stated in the README, for example STBL and VOTE2020 compared to all other assetchains (i.e. in the event of chains using a different komodod).
```

Same komodo version using now for all komodo assetchains - confusing information inreadme because of need to use different komodod commits just to grab fresh startup scripts one of the reasons why we moved scripts into dpow repo

Reply:
There should not be system-wide komodod mentioned. There can be different (old) version copy-pasted in usr/bin and problem would be hard to catch.
Most ops have custom scripts and will edit path anyway.